app/scripts/lib/defaults.js

```diff
@@ -4,5 +4,5 @@ module.exports = {
     username: 'jbloggs',
     hoursPerDay: 7.5,
     useStartDate: false,
-    startDate: '2000-01-01'
+    startDate: new Date().toISOString().substring(0, 10)
```

Comment:
I was concerned that this might be a problem since it gets evaluated whenever settings are saved or restored, but it's inexpensive. I was also worried that users without a start date would have this set instead every time and we wouldn't want that for sync'd storage, but it isn't because it'll be fixed on first use.

I'd argue that Jan 1st seems safer, but actually someone configuring this for the first time likely has a start date closer to today than to Jan 1st!